### PR TITLE
TG Brain Trauma Rework Port

### PR DESCRIPTION
**Pull request by scrubmcnoob**
Ports:
tgstation/tgstation#31727
tgstation/tgstation#33405
tgstation/tgstation#36227
tgstation/tgstation#33378
tgstation/tgstation#33506

These are all the subsequent fixes for brain trauma that aren't just adding in new phobias or tin foil hats.

:cl:XDTM, ported by scrubmcnoob
add: Brain damage has been completely reworked!
remove: Brain damage now no longer simply makes you dumb. Although most of its effects have been shifted into a brain trauma.
add: Every time you take brain damage, there's a chance you'll suffer a brain trauma. There are many variations of brain traumas, split in mild, severe, and special.
add: Mild brain traumas are the easiest to get, can be lightly to moderately annoying, and can be cured with mannitol and time.
add: Severe brain traumas are much rarer and require extensive brain damage before you have a chance to get them; they are usually very debilitating. Unlike mild traumas, they require surgery to cure. A new surgery procedure has been added for this, the aptly named Brain Surgery. It can also heal minor traumas.
add: Special brain traumas are rarely gained in place of Severe traumas: they are either complex or beneficial.
However, they are also even easier to cure than mild traumas, which means that keeping these will usually mean keeping a mild trauma along with it.
add: Mobs can only naturally have one mild trauma and one severe or special trauma.
balance: Brain damage will now kill and ruin the brain if it goes above 200. If it somehow goes above 400, the brain will melt and be destroyed completely.
balance: Many brain-damaging effects have been given a damage cap, making them non-lethal.
del: The Unintelligible mutation has been removed and made into a brain trauma.
del: Brain damage no longer makes using machines a living hell.
add: Abductors give minor traumas to people they experiment on.
/:cl:

### CODE REVIEW

File: strings/traumas.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
         "@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH",
         "without oxigen blob don't evoluate?",
         "@pick(semicolon)CAPTAINS A COMDOM",
-        "@pick(semicolon)@pick(traitor) @pick(george) @pick(mellens) is grifing me HALP!!!",
+        "@pick(semicolon)@pick(faggot_traitor) @pick(george) @pick(mellens) is grifing me HALP!!!",
```
Comment (DarkPyrolord) [inline strings/traumas.json]: Might as well point this out before it passes the Turing test :P

File: strings/traumas.json [hunk 2]
```diff
@@ -43,6 +44,7 @@
         "geT THE FUCK OUTTTT",
         "H U G B O X",
         "@pick(semicolon)CRASHING THIS STTAYTION WITH NIO SURVIVROS",
+        "@pick(servers) is down!! @pick(bug)",
```
Comment (DarkPyrolord) [inline strings/traumas.json]: Same here

File: strings/traumas.json [hunk 3]
```diff
@@ -53,14 +55,14 @@
         "@pick(semicolon)I'VE GOT BALLS OF STEEL",
         "NO I'M ONNA KILL YOU MOTHERFUCKER OLD STYLE",
         "i will snatch erry motherfucker birthday",
-        "SHEDOWLINGS G0T NO ABS!",
+        "@pick(semicolon)the ai and borgs are mettacomming I think",
```
Comment (DarkPyrolord) [inline strings/traumas.json]: :eyes:

Comment (Ziiro) [reply strings/traumas.json]: That brain damage line is a work of art. It's a shame it's too OOC.

**Comment by scrubmcnoob** [comment]
@FlattestGuitar Should be all good.

**Comment by FlattestGuitar** [comment]
Still 'furries' instead of 'ethari'. Please correct that, we don't want people spouting that IC.

**Pull request merged**